### PR TITLE
Add support for apns-collapse-id

### PR DESCRIPTION
**Pull request by Amnell**
Feature #51

Since there has been no activity in PR #52 since Aug 5, 2016 I've decided to contribute with my own implementation of support for apns-collapse-id to hopefully get this merged for public use 👍 

* Persisting of the collapse-id in the savefile
* No collapse-id header will be added if no collapse id is supplied

unspecified-1

### CODE REVIEW

**Comment by blommegard** [comment]
Thanks for you contribution!
I feel bad that I haven't had the time to look at this feature
 yet.
This looks like a good approach, however would love it we could add the ability to save (and repopulate) the text field after a successful push have been made.

What do you think about that?

**Comment by Amnell** [comment]
@blommegard No worries :)
What do you mean with "save (and repopulate) the text field after a successful push have been made"? What should we repopulate the fields with?

**Comment by blommegard** [comment]
My bad, I was mixing it up with `apns-id`. :)

Adding my feedback as a code review, so we can get this merged!

File: Knuff/SBAPNS.h [hunk 1]
```diff
@@ -22,5 +22,6 @@
             toToken:(nonnull NSString *)token
           withTopic:(nullable NSString *)topic
            priority:(NSUInteger)priority
+         collapseID:(NSString *)collapseID
```
Comment (blommegard) [inline Knuff/SBAPNS.h]: We are missing nullability type here

**Comment by blommegard** [comment]
@Amnell You could give me push right to your fork and I could fix the issues there if you dont have the time as well. :)

**Comment by Amnell** [comment]
Sorry @blommegard, I must've missed the notifications from Github. Fixed now

**Pull request merged**